mock-core-configs/mock-core-configs.spec

```diff
@@ -98,7 +102,7 @@ fi
 if [ -s /etc/mageia-release ]; then
     mock_arch=$(sed -n '/^$/!{$ s/.* \(\w*\)$/\1/p}' /etc/mageia-release)
 else
-    mock_arch=$(python3 -c "import dnf.rpm; import hawkey; print(dnf.rpm.basearch(hawkey.detect_arch()))")
+    mock_arch=$(%{python3} -c "import dnf.rpm; import hawkey; print(dnf.rpm.basearch(hawkey.detect_arch()))")
```

Comment:
Ok, I had to test this;  the `%python3` macro is baked into the binary RPM scriptlet expanded (at build time).  Therefore, at installation time, there's no need to have python-srpm-macros installed.

Reply:
So there shouldn't be any problems with this solution?

Reply:
Just verified for myself, looks good to me.